vi.lproj/Localizable.strings

```diff
@@ -194,18 +194,18 @@
 
 "SLEEP_TIMER_MENU" = "Hẹn giờ tắt";
 
-"SLEEP_TIMER_MENU_DISABLED" = "Đã tắt";
+"SLEEP_TIMER_MENU_DISABLED" = "Chưa hẹn";
 "SLEEP_TIMER_MENU_1" = "1 phút";
 "SLEEP_TIMER_MENU_15" = "15 phút";
 "SLEEP_TIMER_MENU_30" = "30 phút";
 "SLEEP_TIMER_MENU_60" = "60 phút";
 
-"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "tên";
-"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "địa chỉ";
-"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "ảnh artwork";
+"EDIT_STATION_WINDOW_NAME_FIELD_LABEL" = "Tên đài";
+"EDIT_STATION_WINDOW_URL_FIELD_LABEL" = "Địa chỉ";
+"EDIT_STATION_WINDOW_IMAGE_FIELD_LABEL" = "Ảnh đại diện";
```

Comment:
In English, I'm leaving these without capitalization, which is how the Music app shows its metadata/info panel in macOS 11. I'm happy either way, though!